flexget/plugins/sites/serienjunkies.py

```diff
@@ -166,7 +166,7 @@ def find_all_titles(self, search_title):
     def check_language(self, languages):
         # Cut additional Subtitles
         try:
-            languages = languages[:languages.index("+")]
+            languages = languages.split('+', 1)[0]
```

Comment:
It only fixes the issue because it hides it. `languages.split('+', 1)[0]` will always return something. If there is no `+` in the string, then it simply returns the whole thing.